workspace/popjava/src/popjava/dataswaper/POPByte.java

```diff
@@ -0,0 +1,76 @@
+package popjava.dataswaper;
+
+import popjava.buffer.POPBuffer;
+/**
+ * Primitive settable byte for POPJava, needed for {@link popjava.annotation.POPParameter} direction use in methods 
```

Comment:
I do not undersand the use-case of this. From my knowledge, normal integers (int) should work correctly with the direction annotation